src/desktop.md

```diff
@@ -357,14 +357,36 @@ impact your distributable application.
 
 #### Linux
 
+The executable can be found in your project under `build/linux/<build mode>/bundle/`. Alongside your 
+executable binary in the `bundle` directory there are two directories, `lib` which contains required `.so`
```

Comment:
Nit: s/,/:/

Reply:
Refactored the two points into a list, following the new `:` in the text.